lib/commands/file-movement.js

```diff
@@ -103,7 +100,7 @@ function isDocumentsContainer(containerType) {
 async function createService(remotePath) {
   if (CONTAINER_PATH_PATTERN.test(remotePath)) {
     const {bundleId, pathInContainer, containerType} = await parseContainerPath.bind(this)(remotePath);
-    const service = await createAfcClient(this.device.udid, bundleId, containerType);
+    const service = await createAfcClient(this.device.udid, bundleId);
     const relativePath = isDocumentsContainer(containerType)
```

Comment:
do we still need this check?

Reply:
i left it for backwards compatibility, in the past the lowercase documents container worked, to be able tot support this i left the path replacement in which is replacing `documents` with `Documents`